autoscale/src/main/java/com/sequenceiq/periscope/controller/DistroXAutoScaleClusterV1Controller.java

```diff
@@ -58,91 +66,102 @@
     @Inject
     private CloudbreakMessagesService messagesService;
 
+    @Inject
+    private TimeAlertRequestConverter timeAlertRequestConverter;
+
+    @Inject
+    private LoadAlertRequestConverter loadAlertRequestConverter;
+
+    @Inject
+    private EntitlementValidationService entitlementValidationService;
+
     @Override
-    @CheckPermissionByAccount(action = AuthorizationResourceAction.DATAHUB_READ)
+    @CheckPermissionByAccount(action = AuthorizationResourceAction.SCALE_DATAHUB)
     public List<DistroXAutoscaleClusterResponse> getClusters() {
         return distroXAutoscaleClusterResponseConverter.convertAllToJson(asClusterCommonService.getDistroXClusters());
     }
```

Comment:
getClusters is still an AccountLevel Permission Check.

Reply:
i think scaledatahub right will be present only in resource roles, so it wont work with account level, so this will work only with poweruser, so it is better to change this `@DisabledCheckPermissions` since we decided to disable authz on list for now, right @lnardai ?

Reply:
ok added `@DisabledCheckPermissions` for listApi in sync with other listApi like distrox.